data/macos.yaml

```diff
@@ -724,6 +724,20 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#System_Preferences'
 ---
+name: MacOSUnifiedLogFiles
+doc: Unified logging files
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '/var/db/diagnostics/*'
```

Comment:
Under this folder there are several different file formats/types, the unified log ones are the .tracev3 (binary). In here you will have also the statistics file (e.g. logdata.statistics.#.txt). Not sue if you want to split those here and point only specifically to the actual log file. In case consider changing it to 
'/var/db/diagnostics/*.tracev3'